.github/workflows/WorkflowHook.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           recreate: true
           message: >
             Hi @${{ github.event.workflow_run.actor.login }}!
-            Codes seem to be unformatted. To resolve this issue, please run `./gradlew spotlessKotlinApply` and fix the results of ./gradlew lintDebug..
+            Codes seem to be unformatted. To resolve this issue, please run `./gradlew --auto-correct reportMerge --continue` and fix the results of ./gradlew lintDebug..
```

Comment:
Maybe we have to use spotlessKotlinApply for now? 👀

Reply:
Fixed~ e0a4772